Source/Csla.Blazor/CslaValidator.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.AspNetCore.Components;
+using Microsoft.AspNetCore.Components.Forms;
+
+namespace Csla.Blazor
+{
+
+  /// <summary>
+  /// Class offering integration of CSLA validation into Blazor edit forms
+  /// This is a component with no visible behaviour; it simply requests hook up of validation
+  /// </summary>
+  public class CslaValidator : ComponentBase
+  {
+    
+    [CascadingParameter]
+    public EditContext CurrentEditContext { get; set; }
```

Comment:
Is this the name the Blazor community has adopted? Not BindingContext or DataContext from the various XAML dialects?

Reply:
EditContext is the name of the built in class they have created for cascading down from their EditForm component, and CurrentEditContext is the name of the parameter in the built in DataAnnotationsValidator component on which mine is based.

It's consistent and interoperable with what the ASP.NET team have done; they seem to have chosen their own route here.